package.json

```diff
@@ -39,18 +39,18 @@
   "dependencies": {
     "@types/mdast": "^3.0.3",
     "@types/unist": "^2.0.3",
-    "html-to-react": "^1.3.4",
+    "html-to-react": "^1.4.5",
     "mdast-add-list-metadata": "1.0.1",
     "prop-types": "^15.7.2",
-    "react-is": "^16.8.6",
+    "react-is": "^17.0.0",
     "remark-parse": "^9.0.0",
     "unified": "^9.0.0",
     "unist-util-visit": "^2.0.0",
     "xtend": "^4.0.1"
   },
   "peerDependencies": {
     "@types/react": "^15.0.0 || ^16.0.0",
```

Comment:
/cc @rexxarsm, @ChristianMurphy I guess types should be unlocked too?

Btw: I’m going to unlock right now to at least hide the warnings. Tests seem to indicate it’s all fine, but then this PR can land later when enzyme updates